ext/lib/mgmt/mcumgr/cmd/fs_mgmt/pkg.yml

```diff
@@ -0,0 +1,27 @@
+#
```

Comment:
Is it possible to avoid imprinting of all pkg.yml as this files are mynewt relevant?